policy/Examples.md

```diff
@@ -0,0 +1,788 @@
+  "prev_policies": null,
+  "rules": [
+    {
+      "name": "Valid Provider Drop Offs",
+      "rule_id": "7a043ac8-03cd-4b0d-9588-d0af24f82832",
+      "rule_type": "count",
+      "geographies": [
+        "6dc968c7-19f4-421c-b9d1-683dd3cdb632",
+        "2a4fbdb9-ff76-4060-aa92-1d37e26732e8",
+        "9bb19cd1-2530-4f7f-8de0-80e7326a3e32",
+        "fe9c910a-7aca-4a42-9d63-e014b3c243d7",
+        "7beb1d83-66e7-4654-8c6b-6710fa26d1bd",
+        "c7553640-730f-4ae1-a422-68bac4b849cc",
+        "e42f7e97-b5e6-4ebe-8ddc-05fc806ce54e",
+        "b539054b-541a-43b3-a182-58a0bd0958fd",
+        "73779ce8-e0fb-48c0-96ba-a1e7f7738279",
+        "aa4dc424-09e4-48f3-8471-df5186927016",
+        "f5f4a15d-447f-4969-aedb-a0e94ae5b183",
+        "456c25f0-a9ce-4ff3-8610-3cee919a3539",
+        "0a484e09-7a95-4e7d-86c7-a10a58268ee2",
+        "06b4e69e-da53-4340-8354-5a2262034657",
+        "b1fdf441-ce46-4f22-bb70-dd2e99df1001",
+        "2166b7dd-10ab-4219-9921-0d8c0f082308",
+        "86f9a2bd-48c8-4447-b6eb-60916da16aa1",
+        "d5d889c5-b6b9-4b83-bbcb-f5209d8dbcc3",
+        "5a5b5ffa-5f9f-4db8-ba09-72c5deaac41a",
+        "8ce201f3-34d7-46a2-aed3-282fcb6938ac",
+        "2d7f76f0-f45e-4563-8be1-280f77b1181a",
+        "9912fa40-b594-492f-91d0-113a7568bb2b"
+      ],
+      "statuses": {
+        "available": [
+          "provider_drop_off"
+        ]
+      },
+      "vehicle_types": [
+        "bicycle",
+        "scooter"
+      ],
+      "maximum": 0
+    },
+    {
+      "name": "Drop-off No-Fly Zones",
+      "rule_id": "596d7fe1-53fd-4ea4-8ba7-33f5ea8d98a6",
+      "rule_type": "count",
+      "geographies": [
+        "e0e4a085-7a50-43e0-afa4-6792ca897c5a"
+      ],
+      "statuses": {
+        "available": [
+          "provider_drop_off"
+        ]
+      },
+      "vehicle_types": [
+        "bicycle",
+        "scooter"
+      ],
+      "maximum": 0
+    }
+  ]
+}
+```
+
+#### Walkthrough
+
+##### Abstract
+
+The Venice Special Operations Zone Restrictions published by LADOT !FIXME NEED LINK! specifies a series of valid drop off points within a greater overarching no-dropoff zone.
+
+##### Policy Design Approach
+
+The Venice Special Operations Zone Policy consists of two rules, first, a rule which contains geographies representing the 'valid' `provider_drop_off` zones, and a second rule which has the overarching Venice geography. During the evaluation of the policy, a set of vehicles will be provided, we'll refer to this set as `V`. The first rule will match all vehicles which have been `provider_drop_off`'d within one of the given geographies, let's call this group of vehicles `v_1`. When evaluating the second rule, the vehicles which will be considered in the evaluation will be a subset `V \ v_1`. At this point, we are considering a subset which cannot have a vehicle in a valid drop off zone for this policy; and because of this, all vehicles that match with the second rule will be considered in violation of the policy.
```

Comment:
This explanation of the above rules doesn't make sense to me and it's especially confusing since the JSON of each rule seems essentially identical. Why wouldn't the geography in the 2nd rule just be listed among those in the first rule? And this explanation makes me wonder if somehow the order of the rules are important and should be combined together to produce some kind of hybrid rule.

Reply:
The rule order is important (per the spec).  The JSON is similar because it applies to the same types of vehicles and operations (`provider_drop_off`), but different because some of the specified areas are permitted and others aren't.

Reply:
The goal of the Policy schema and its corresponding Compliance engine is to express as many different Policy concepts as possible, with a bare minimum of abstractions and exceptions.

Reply:
Ah, I didn't see that section about how the order of the rules is important. My bad. 

However, it's still not clear how to determine which areas are permitted and which ones aren't because it appears that the only data available in the geographies/{id} endpoint is the id of the geography.